examples/calc_flatfield.py

```diff
@@ -0,0 +1,103 @@
+"""
+Extract flat field coefficients from flasher data files.
+"""
+from traitlets import Dict, List, Unicode
+
+from ctapipe.core import Provenance
+from ctapipe.io import HDF5TableWriter
+from ctapipe.core import Tool
+from ctapipe.io import EventSourceFactory
+
+from ctapipe.image import ChargeExtractorFactory, WaveformCleanerFactory
+from ctapipe.calib.camera.flatfield import FlatFieldFactory
+from ctapipe.io.containers import MonitorDataContainer
+
+
+class FlatFieldGenerator(Tool):
```

Comment:
Generator has a very specific meaning in Python. This class is not a Generator in that sense.
It would be better to rename it. (maybe to  `FlatFieldHDFWriter`? )